### PR TITLE
ui: move profile add buttons

### PR DESCRIPTION
**Pull request by Forfold**
Part of #2693

Makes the create buttons for the profile page more visible when the browser width is large. On mobile, the two add buttons will remain as a FAB button in the bottom right.

Also updates the secondary action IconButton to be `color='secondary'` for `OtherActions`, and cleans up passing a prop through a couple components that can be pulled from an existing hook call.

Dark mode example

Screen Shot 2022-11-29 at 12 19 17 PM

Light mode example w/ notification rule add disabled when no contact methods are present

Screen Shot 2022-11-29 at 1 33 39 PM

### CODE REVIEW

File: web/src/cypress/integration/profile.ts [hunk 1]
```diff
@@ -13,7 +14,12 @@ function countryCodeCheck(
     const name = 'CM SM ' + c.word({ length: 8 })
     const type = c.pickone(['SMS', 'VOICE'])
 
-    cy.pageFab('Contact')
+    if (screen === 'mobile') {
+      cy.pageFab('Add Contact Method')
```
Comment (Forfold) [inline web/src/cypress/integration/profile.ts:18]: I considered having a second param to pageFab called "fallbackSelector" if a page fab isn't present. It just prioritizes mobile at that point. Thoughts?

Comment (mastercactapus) [reply web/src/cypress/integration/profile.ts:18]: I think in this case doing the switch without the indirection is clearer.

The UI itself splits, and the test code reflects that. On wide, there's no FAB so calling `pageFab` would probably be confusing without context.

Comment (Forfold) [reply web/src/cypress/integration/profile.ts:18]: Yeah that's pretty much where I landed. Sounds good 👍

File: web/src/cypress/integration/profile.ts [hunk 2]
```diff
@@ -54,13 +60,15 @@ function testProfile(): void {
...
-          })
-          .engineTrigger()
-          .then(() => svc.id)
+        return (
+          cy
+            .createEPStep({
+              epID: svc.epID,
+              targets: [{ type: 'user', id: profile.id }],
+            })
+            // .engineTrigger()
```
Comment (tony-tvu) [inline web/src/cypress/integration/profile.ts:69]: Should we remove this comment?

**Review by tony-tvu — DISMISSED** [review]
LGTM!

File: web/src/cypress/integration/profile.ts (hunk 2 above)
Comment (Forfold) [reply web/src/cypress/integration/profile.ts:69]: Oh, this was causing tests to fail for me locally, but I don't think it was for Caza. I'll uncomment out and see what happens on CI. It also looks like there's a playwright test I need to update as well

Comment (mastercactapus) [reply web/src/cypress/integration/profile.ts:69]: This is needed, so the test doesn't hang waiting for the engine to run in the background

**Review by tony-tvu — APPROVED** [review]

**Review by mastercactapus — APPROVED** [review]

**Pull request merged**